### ISSUE TITLE
Bad stitch prediction results

### ISSUE THREAD

**Issue by zycliao** [maria-korosteleva/Garment-Pattern-Estimation#3]
Hi, thanks for the amazing work. I'm trying to test the pre-trained model on the official dataset. However, the result looks bad. I'm wondering if I did anything wrong.

Basically, I used the pre-trained weights and tested on a few garments provided by the [dataset](https://doi.org/10.5281/zenodo.5267549). I ran `predict_per_example.py` with the default configurations. For example, when I tested `garment_pattern/dress_150/dress_2Q19HWLQ83/dress_2Q19HWLQ83_sim.obj`, the result was as below,
![image](https://user-images.githubusercontent.com/25116439/221568335-b4e84e73-8b13-4e60-b044-b8f9b5335398.png)
but after importing it into Garment Viewer in Maya, the result was
![image](https://user-images.githubusercontent.com/25116439/221568183-bb939dfe-b633-4333-97f1-62bfa1434cab.png)

**Comment by maria-korosteleva** [comment]
Hi @zycliao, thank you for your interest in our work!

1. Could you specify what exactly is not meeting your expectations? 
    The shape of the panels is close to GT, and they are placed +/- correctly relative to the body. Stitches are quite bad, but, as reported in the paper, the stitch prediction on unseen types is, unfortunately, not ideal:
    <img width="557" alt="image" src="https://user-images.githubusercontent.com/6837635/222215474-59c57e3f-a716-4343-9359-bdc2219a8e7c.png">
  So, I would say that the presented example would match my expectations 🤔

    For reference, this is the GT pattern from a garment in question 
    ![dress_2Q19HWLQ83_pattern](https://user-images.githubusercontent.com/6837635/222217100-ac5bb553-b504-4ff0-810d-01d236b71f72.png)

2. To double-check if your setup is correct, may I ask you to run the evaluation script to see if the accuracy numbers on your installation are similar to the ones reported in the paper? The command is as follows, running from the directory root:
   ```
   python nn/evaluation_scripts/on_test_set.py -sh models/att/att.yaml -st models/att/stitch_model.yaml --unseen 
   ```
   Please, report the numbers that you will get

**Comment by zycliao** [comment]
@maria-korosteleva Thanks for your reply. I was mainly concerned about the stitch (the blue lines in my screenshot). They look totally different from GT. And after the simulation, it becomes a mess.

**Comment by maria-korosteleva** [comment]
Yes, please, report the results on stitches accuracy from the evaluation script, as I requested above, to double-check that the setup is correct

If you have a need to use this specific prediction in the downstream application, you can adjust (add/remove) the stitches manually following Qualoth's manual

**Comment by jinleyang** [comment]
@zycliao Hi! I have encountered the exact same issue as you. Do you have any ideas about it?

**Comment by jinleyang** [comment]
> @zycliao Hi! I have encountered the exact same issue as you. Do you have any ideas about it?

Specifically, when I use the same testing data (for example, dress_sleeveless_2550/dress_sleeveless_6VN3H6FMOY in seen test split), the stitch results are totally different using predict_per_example.py compared with using nn/evaluation_scripts/on_test_set.py. The one by predict_per_example.py is quite bad (always with one stitch only per garment), but the one by given evaluation script on_test_set.py is quite good (looks correct when visualizing in 3D). By the way, the shape prediction of both script are both good, only stitch is influenced.
